src/main/kotlin/org/rust/lang/utils/RsDiagnostic.kt

```diff
@@ -696,6 +693,19 @@ sealed class RsDiagnostic(
             fixes = listOf(ConvertToReferenceFix(element), ConvertToBoxFix(element))
         )
     }
+
+    class ExperimentalFeature(
+        element: PsiElement,
+        private val presentableFeatureName: String,
+        private val fix: AddFeatureAttributeFix? = null
+    ) : RsDiagnostic(element) {
+        override fun prepare(): PreparedAnnotation = PreparedAnnotation(
+            ERROR,
+            E0658,
+            header = escapeString("$presentableFeatureName is experimental"),
```

Comment:
We don't really need `escapeString` here?

Reply:
`presentableFeatureName ` can potentially contain symbols which we need to escape